src/gsi/setuprw.f90

```diff
@@ -648,10 +651,18 @@ subroutine setuprw(obsLL,odiagLL,lunin,mype,bwork,awork,nele,nobs,is,conv_diagsa
      endif
 
 !    adjust obs error for TDR data
-     if(data(iobs_type,i) > three .and. ratio_errors*error > tiny_r_kind &
+     !if(data(iobs_type,i) > three .and. ratio_errors*error > tiny_r_kind &
```

Comment:
Another file, I don't think this should be in the netCDF change PR.

Reply:
In another PR.